dbt_coves/tasks/generate/airflow_dags.py

```diff
@@ -38,6 +38,12 @@ def register_parser(cls, sub_parsers, base_subparser):
             required=False,
             help="Folder where YML files will be read from",
         )
+        subparser.add_argument(
+            "--dag-destination",
```

Comment:
If the other param is called `--from-path`, should this be named `--to-path` / `--target-path`?

Reply:
`--target-path` seems fine